### ISSUE TITLE
[a11y review] Supply reordering: Make continue-progress options on start screen accessible

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#102068]
> [!NOTE]
> **QUICK SUMMARY:** The Medical Device Ordering tool is moving from buttons to CTA links in the 'continue in progress' alert on the form's landing page. The Cartography team requested a11y feedback on the move from buttons to CTA links.

## Issue Description
The Medical Device Ordering tool is moving from buttons to CTA links in the 'continue in progress' alert on the form's landing page. The Cartography team requested a11y feedback on the move from buttons to CTA links.

## About the Team

- **Team name**: Medical Device Ordering
- **Product name:** Medical Device Ordering
- **Product manager:**  
- **Slack Channel:**

### Supporting Artifacts 
- Ticket linked below

```[tasklist]
### a11y Tasks
- [x] Review the ticket and provide a11y feedback on the proposed design
```
```[tasklist]
### Acceptance Criteria
- [x] Feedback has been provided in the ticket for the team
- [ ] Validate that the team doesn't have further questions about the feedback
```
```[tasklist]
## Ticket Updates (CAIA Internal)
- [x] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Naomi)
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
```

```[tasklist]
### Related Tickets
- [ ] #101103 
```

**Comment by rileyorr** [comment]
I've provided feedback in the related ticket and will wait for confirmation from the team that they don't have followup questions prior to moving this to validate!

**Comment by rileyorr** [comment]
@lakper and @sara-amanda this has been ready to close for a while now - should I just move it to close? Wasn't sure if we're still following the previous process where the team moves tickets to validate and y'all review and close the ticket
